lib/content-services/src/lib/tag/tag-node-list.component.html

```diff
@@ -1,9 +1,19 @@
-<mat-chip-list>
-    <mat-chip class="adf-tag-chips"
-              *ngFor="let currentEntry of tagsEntries; let idx = index" (removed)="removeTag(currentEntry.entry.id)">
-        <span id="tag_name_{{idx}}">{{currentEntry.entry.tag}}</span>
-        <mat-icon *ngIf="showDelete" id="tag_chips_delete_{{currentEntry.entry.tag}}"
-                  class="adf-tag-chips-delete-icon" matChipRemove>cancel
-        </mat-icon>
-    </mat-chip>
-</mat-chip-list>
+<div class="adf-tag-node-list" [class.adf-flex-column]="limitTagsDisplayed && (!calculationsDone || columnFlexDirection)" #nodeListContainer>
+    <mat-chip-list [class.adf-full-width]="limitTagsDisplayed && !calculationsDone">
+        <mat-chip class="adf-tag-chips"
+                  *ngFor="let currentEntry of tagsEntries; let idx = index"
+                  (removed)="removeTag(currentEntry.entry.id)">
+            <span id="tag_name_{{idx}}">{{currentEntry.entry.tag}}</span>
```

Comment:
Do we use somewhere _ for ids? I think more ofter we use -.

Reply:
Yes we do, the id was already defined when I started changing the component and I'm wondering if this would be a breaking change for someone who already started using this component, if we can change without breaking anything than I would do it, @DenysVuika what do you think?

Reply:
I don't think we have any special requirements for the ID format, except it should be unique